Documentation/General/finalReport/InteractionDesign.tex

```diff
@@ -1,2 +1,106 @@
-\chapter{Human-Computer Interaction }
+\chapter{Human-Computer Interaction}
+
+\section{Introduction}
+This section describes the interaction between the user and the computer and how we measured this.
+The user is very important for us, the user is the one that should use the application.
+Our system is build for interaction between users and is improving by user interaction.
+The interaction between users is the chatroom that our whole music application is build around.
+The music itself is based on valence and arousal witch is classified by the user.
```

Comment:
I would not go into this detail for the HCI chapter. Just note that the music suggestions are based on the current / specified mood of the user.